internal/storage/payload_reader.go

```diff
@@ -96,6 +97,23 @@ func (r *PayloadReader) GetBoolFromPayload() ([]bool, error) {
 	return ret, nil
 }
 
+// GetBoolFromPayload2 returns bool slice from payload.
+func (r *PayloadReader) GetBoolFromPayload2() ([]bool, error) {
```

Comment:
since the performance result is impressive, I would request to remove payload v1 and rename  GetBoolFromPayload2 to GetBoolFromPayload

Reply:
Done